ui/appui-layout-react/src/appui-layout-react/widget-panels/Grip.scss

```diff
@@ -25,6 +25,16 @@ $line-grip-width: calc(#{$nz-grip-width} - 2px);
     transform: translate(-50%, -50%);
   }
 
+  &.nz-left {
+    transition-property: background-color, right;
+  }
+  &.nz-right {
+    transition-property: background-color, left;
+  }
+  &.nz-top {
+    transition-property: background-color, bottom;
+  }
```

Comment:
How about `nz-bottom`?

Reply:
This is also not adding a `delay`, but a `duration` to "position properties"? I don't see where we would change one of `right/left/bottom` properties though for the transition to kick in?

Reply:
1. `nz-bottom` is not changing its `top` like this on `nz-collapsed`, and anyway this issue really only occured on the right panel, but I'm adding safeguard in case it would ever happen in the other ones.
2. I'm not adding a `delay` but simply a duration to these properties because that is enough to fix the issue (adding a delay would get weird as the blue bar wouldnt appear and the panel would move around.
3. This property is actually changed in https://github.com/iTwin/appui/blob/2af8649373b55aedc4cd05d59f507243ecdf9eb2/ui/appui-layout-react/src/appui-layout-react/widget-panels/Panel.scss#L107 (and other lines for other sides), but the `transition` needs to be there before the `nz-collapsed` is added, so I cant put it in the same place for this to work correctly.

Reply:
Well this new CSS should be added to `Panel.scss` OR the `right: 0;` (and related CSS) should be moved to `Grip.scss`.

Reply:
Approving for now just to get the fix out, but would be great to add a todo at least..